nautobot/ui/src/views/generic/ObjectCreate.js

```diff
@@ -18,14 +20,22 @@ export default function GenericObjectCreateView({ list_url }) {
     const { data, error } = useSWR(list_url, fetcher);
 
     if (!app_name || !model_name) {
-        return <></>;
+        return <GenericView />;
```

Comment:
We might want to consider `react-router` layout nesting not to duplicate so many `<GenericView>` usages. It is a feature invented exactly for this kind of layouts, but I didn't want to implement it in this PR to focus on just one thing at a time.

Reply:
👍 When I was looking at how we will do the login screen I figured we'll have to do just that.